crates/uv-cli/src/lib.rs

```diff
@@ -691,6 +693,103 @@ pub enum VersionBump {
     Dev,
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord)]
+pub struct VersionBumpSpec {
+    pub bump: VersionBump,
+    pub value: Option<u64>,
+}
+
+impl std::fmt::Display for VersionBumpSpec {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self.value {
+            Some(value) => write!(f, "{}={value}", self.bump),
+            None => self.bump.fmt(f),
+        }
+    }
+}
+
+impl std::str::FromStr for VersionBumpSpec {
+    type Err = String;
+
+    fn from_str(input: &str) -> Result<Self, Self::Err> {
+        let (name, value) = input
+            .split_once('=')
+            .map_or((input, None), |(name, value)| (name, Some(value)));
+
```

Comment:
I personally find `.map().unwrap_or()` easier to read

Reply:
Yeah this reads more clear to me, changed this up.